pkgs/applications/misc/notejot/default.nix

```diff
@@ -41,6 +42,13 @@ stdenv.mkDerivation rec {
     libgee
   ];
 
+  patches = [
+    (fetchpatch {
```

Comment:
Explain why the patch is needed - and if it can be removed in the next release.

Reply:
Sorry, forgot to leave a comment. Done.